### PR TITLE
Swagger apiview 

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

**Comment by JimSuplizio** [comment]
@ruowan I don't think @Azure/azure-sdk-eng are the right reviewers here, that's the engineering system team and these aren't engineering system changes. You should really have the appropriate person(s) from the Java team review these changes.

File: tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerTypes.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace swagger_api_parser
```
[the remark this answers is not in the document]
Comment (ruowan) [reply tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerTypes.cs]: Done. Rename to SwaggerApiParser

File: tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerCodeFileMerger.cs [hunk 2]
```diff
@@ -2,180 +2,208 @@
...
 
- public class SwaggerCodeFileMerger
+public class SwaggerCodeFileMerger
+{
+    private readonly Dictionary<string, CodeFile> originalResult;
+
+    public SwaggerCodeFileMerger()
     {
-        private readonly Dictionary<string, CodeFile> originalResult;
+        this.originalResult = new Dictionary<string, CodeFile>();
```
[the remark this answers is not in the document]
Comment (ruowan) [reply tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerCodeFileMerger.cs]: This file will be deprecated.

File: tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerAPIViewTypes.cs [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using ApiView;
+using APIView;
+
+
+namespace swagger_api_parser
+{
```
[the remark this answers is not in the document]
Comment (ruowan) [reply tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerAPIViewTypes.cs]: Done.

File: tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerTypes.cs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+{
+    public class SwaggerSpec
+    {
+        public string swagger { get; set; }
```
[the remark this answers is not in the document]
Comment (ruowan) [reply tools/apiview/parsers/swagger-api-parser/swagger-api-parser/SwaggerTypes.cs]: Yes.

File: tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/docs/deviceUpdateCurrentView.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
+###  `/deviceupdate/{instanceId}/updates`
```
Comment (praveenkuttappan) [inline tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/docs/deviceUpdateCurrentView.md:1]: Let's remove this file

File: tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/docs/deviceUpdateOperationJsonView.md [hunk 6]
```diff
@@ -0,0 +1,330 @@
+###  `/deviceupdate/{instanceId}/updates`
```
Comment (praveenkuttappan) [inline tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/docs/deviceUpdateOperationJsonView.md:1]: Same with this file

**Review by praveenkuttappan — APPROVED** [review]
Looks good. Only change required is to delete some stale files. for e.g.
`swagger-api-parser/SwaggerApiParser/docs/images/*`

**Review by chidozieononiwu — COMMENTED** [review]
Should this PR be requeued against main, since `feature/apiview_code_group` is now stale

**Pull request closed without merging**